test/Microsoft.Azure.WebJobs.Host.FunctionalTests/HostListenerFactoryTests.cs

```diff
@@ -25,17 +26,45 @@ namespace Microsoft.Azure.WebJobs.Host.UnitTests.Listeners
     public class HostListenerFactoryTests
     {
         private readonly IJobActivator _jobActivator;
-        private IConfiguration _configuration = new ConfigurationBuilder().Build();
+        private readonly IHost _testHost;
+        private IConfiguration _configuration;
 
         public HostListenerFactoryTests()
         {
-            var hostBuilder = new HostBuilder();
```

Comment:
I refactored these tests to use a proper builder because I need to test the ordering of the default decorators added by our services DI. The test below ensures that when a user adds their own decorators, they're ordered correctly with built-ins.